packages/ui/src/accounts/components/AccountItem/components/LockItem.tsx

```diff
@@ -80,7 +80,7 @@ export const LockItem = ({
         <BalanceAmount amount={lock.amount} isRecoverable={isRecoverable} />
         {memberInfo && (
           <div>
-            <DetailLabel>Bound to:</DetailLabel>
+            <DetailLabel>{lock.type === 'Voting' ? 'Voted for' : 'Bound to:'}</DetailLabel>
             <MemberInfo member={memberInfo} onlyTop />
```

Comment:
@RafalPloszka this might be correct I can't check rn but are you sure the candidate membership will show here and not the membership of the user who voted ?

Reply:
In VoteLockItem.tsx there is:
`const voteFor = vote?.voteFor?.member`
`const voteForMember = voteFor && asMember(voteFor)`
and `voteForMember` is passed to LockItem `memberInfo` props

So I'm pretty sure, however, this info is not available during Voting Period and if a vote wasn't unsealed. But then `<MemberInfo>` is just not displayed